src/borg/repository.py

```diff
@@ -1123,8 +1107,7 @@ def recover_segment(self, segment, filename):
                 data = data[size:]
 
     def read(self, segment, offset, id, read_data=True):
-        """
-        Read entry from *segment* at *offset* with *id*.
+        """Read entry from *segment* at *offset* with *id*.
 
         If read_data is False the size of the entry is returned instead and integrity checks are skipped.
```

Comment:
there is no error from pep8 checker, so i won't mix functional changes in this PR with pep8 refactoring. i guess we already have all the styles in our source.